extensions/wikia/AdEngine/js/template/roadblock.js

```diff
@@ -0,0 +1,89 @@
+/*global define, require*/
+define('ext.wikia.adEngine.template.roadblock', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.context.uapContext',
+	'ext.wikia.adEngine.provider.gpt.googleSlots',
+	'ext.wikia.adEngine.provider.gpt.helper',
+	require.optional('ext.wikia.adEngine.template.skin'),
+	'wikia.document',
+	'wikia.window',
+	'wikia.log'
+], function (
+	adContext,
+	uapContext,
+	googleSlots,
+	gptHelper,
+	skinTemplate,
+	doc,
+	win,
+	log
+) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.template.roadblock',
+		uapType = 'ruap',
+		medrecSlotEl = doc.getElementById('TOP_RIGHT_BOXAD');
+
+	/**
+	 * Handles medrec slot in case of UAP:roadblock
+	 * @param {Node} medrecSlotEl - medrec slot container element
+	 */
+	function handleMedrec(medrecSlotEl) {
+		var refreshed = false;
+
+		win.addEventListener('adengine.slot.status', function onAdStatusChange(event) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this function anonymous by removing its name: 'function() {...}'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ANamedFunctionExpression)